conjure-java-core/src/test/resources/external-long-test-service.yml

```diff
@@ -11,12 +11,15 @@ types:
     Long:
       external:
         java: java.lang.Long
-    DangerousLongAlias:
-      alias: DangerousLong
-    SafeExternalLongAlias:
-      alias: SafeExternalLong
-    LongAlias:
-      alias: Long
+  definitions:
+    default-package: com.palantir.product
+    objects:
+      DangerousLongAliasEndpoint:
+        alias: DangerousLong
+      SafeExternalLongAliasEndpoint:
+        alias: SafeExternalLong
+      LongAliasEndpoint:
+        alias: Long
```

Comment:
let's rename these to avoid the `Endpoint` suffix since it is very similar to some of the generated code.